packages/grafana-ui/src/components/Collapse/Collapse.tsx

```diff
@@ -145,10 +145,10 @@ export const Collapse = ({
 
   return (
     <div className={panelClass}>
-      <div className={headerClass} onClick={onClickToggle}>
+      <button type="button" className={cx(buttonStyles, headerClass)} onClick={onClickToggle}>
```

Comment:
Seems like a similar button is used more than once, should we have a separate component for it? Maybe a modified version of `IconButton`?

Reply:
@Clarity-89 don't quite understand - do you mean creating a component for `<button type="button ...>`? 🤔

Reply:
Basically, but also with the clear button styles.

Reply:
ah got it. yeah that could have value... 🤔  lemme have a think about implementation

Reply:
gonna merge for now whilst we think about an implementation for exposing an unstyled button.

debating between just exposing a new `Button` variant (`unstyled`), or a whole new component `UnstyledButton`. neither of which feel super nice tbh. wondering whether we _shouldn't_ expose this as although it's being used quite a bit internally, i don't think we'd want to expose it externally as part of `@grafana/ui`? idk, any thoughts? 🤔